src/main/java/org/hyperledger/fabric/sdk/NetworkConfig.java

```diff
@@ -973,7 +1000,7 @@ private CAInfo createCA(JsonObject jsonCA, OrgInfo org) throws NetworkConfigurat
             }
         }
 
-        CAInfo caInfo = new CAInfo(org.mspId, url, regUsers, httpOptions);
+        CAInfo caInfo = new CAInfo(name, url, regUsers, httpOptions);
```

Comment:
Is this change necessary?

Reply:
There was no easy way to pull the CA that we need for the integration tests otherwise.